src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/lynx/insights/quill-services/blots/lynx-insight-blot.ts

```
@@ -0,0 +1,80 @@
+  static nodeValuePropName = '__lynxInsight';
+
+  /**
+   * This custom prop is used on the parent class instead of using 'className'
+   * so that it isn't registered to the wrong child blot type.
+   * This way 'lynx-insight' class can be added to all child insight blots in addition to the 'className'
+   * specified in the child blot classes.
+   */
+  static superClassName = 'lynx-insight';
+
+  static create(value: LynxInsight | LynxInsight[]): HTMLElement {
+    const node = super.create() as HTMLElement;
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)